collect_app/src/androidTest/java/org/odk/collect/android/espressoutils/pages/Page.java

```diff
@@ -161,4 +168,35 @@ public T checkIfAreaWithKeyIsDisplayed(String key) {
         onData(PreferenceMatchers.withKey(key)).check(matches(isDisplayed()));
         return (T) this;
     }
+
+    public T checkIfStringIsEnabled(int stringID) {
+        onView(withText(getInstrumentation().getTargetContext().getString(stringID))).check(matches(not(isEnabled())));
+        return (T) this;
+    }
+
+    public T scrollToElementWithKey(String key) {
+        onData(PreferenceMatchers.withKey(key)).perform(ViewActions.scrollTo());
+        return (T) this;
+    }
+
+    public T checkIfElementWithKeyIsEnabled(String key) {
```

Comment:
The name is checkIfElementWithKeyIsEnabled but in fact you check if it is disabled.